Lecture-6/Task1/l6task1/Application.java

```diff
@@ -0,0 +1,139 @@
+package l6task1;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Application {
+
+    private static String[] coffeeType = {"bean", "ground", "instant"};
+    private static String[] coffeeSort = {"arabica", "robusta"};
+    private static int[] coffeePrice = {10, 20, 30};
+    private static String[] packType = {"can", "bag"};
+    private static int[] packCapacity = {50, 100, 200, 300, 500};
+    private static int[] packValue = {5, 10, 20, 30, 50};
+    private static int[] packPrice = {10, 20, 30, 40, 50};
```

Comment:
What's the difference between `paclValue` and `packPrice`?

Reply:
Got it. Correct name is not value, but `weight`

Reply:
done